lua/metals/commands.lua

```diff
@@ -213,6 +213,16 @@ local commands_table = {
     label = "Update Metals",
     hint = "Update to the latest Metals.",
   },
+  {
+    id = "select_test_suite",
+    label = "Select Test Suite",
+    hint = "Select test suite to run.",
```

Comment:
```suggestion
    hint = "Select the test suite in your workspace to run.",
```
What do you think about expanding it a bit to let the user know that this is choosing from the full workspace?